<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateVersion</class>
 <widget class="QDialog" name="CreateVersion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Versions</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a name and comment for the version to be registered</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="versionName">
      <property name="text">
       <string>Version name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="versionNameLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="comments">
      <property name="text">
       <string>Comments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="commentsTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>541</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="text">
       <string>Register version</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
